src/components/Shared/SocialShareModal.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+// @ts-ignore
+import { Modal } from "react-router-modal";
+import { copyToClipboard } from "lib/util";
+import Tooltip from "rc-tooltip";
+import * as css from "./SocialShareModal.scss";
+
+interface IState {
+  showCopiedFeedback: boolean;
+}
+
+interface IProps {
+  closeHandler: (event: any) => void;
+  url: string;
+}
+
+export default class SocialShareModal extends React.Component<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+
+    this.selectTwitter = this.selectTwitter.bind(this);
+    this.selectReddit = this.selectReddit.bind(this);
+    this.selectFacebook = this.selectFacebook.bind(this);
+    this.selectTelegram = this.selectTelegram.bind(this);
+    this.copyUrl = this.copyUrl.bind(this);
+
+    this.state = {
+      showCopiedFeedback: false,
+    };
+  }
+
+  private showCopiedFeedback(): void {
+    this.setState({ showCopiedFeedback: true });
```

Comment:
elsewhere we use showNotification for this, and those notifications are supposed to go away on their own, but that stopped working at some point... would be good to still use that unified notification system though

Reply:
The spec specifically asked for this design, not a notification.